tests/src/test/scala/actionContainers/BasicActionRunnerTests.scala

```diff
@@ -309,6 +310,7 @@ trait BasicActionRunnerTests extends ActionProxyContainerTestUtils {
       "api_key" -> "abc",
       "namespace" -> "zzz",
       "action_name" -> "xxx",
+      "action_version" -> "0.0.1",
```

Comment:
the tests in the runtimes will need to adapt for this change - for example:

https://github.com/apache/openwhisk-runtime-nodejs/blob/314cd92ea0c6681ba6455262f17fa9f5b777fbad/tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala#L75-L88

https://github.com/apache/openwhisk-runtime-php/blob/6cac63420c9bb1b9768c91d8cd384244a76fad2a/tests/src/test/scala/runtime/actionContainers/Php7ActionContainerTests.scala#L102-L119

https://github.com/apache/openwhisk-runtime-dotnet/blob/ef821d18ef6a96696bed521212333224575d8df6/tests/dotnetshared/Environment.cs#L24-L34

cc @shawnallen85 @akrabat  @sciabarracom

Reply:
All runtimes now have PRs to deal with this change, no need to make it optional.